docs/pages/deploy-a-cluster/deployments/aws-ha-autoscale-cluster-terraform.mdx

```diff
@@ -163,8 +163,7 @@ AMIs when we release a new version of Teleport. The AMI names follow the format:
 where `<type>` is either `oss` or `ent` (Enterprise), `<version>` is the version of Teleport, e.g. `(=teleport.version=)`,
 and `<arch>` is either `x86_64` or `arm64`.
  
-FIPS 140-2 compatible AMIs (which deploy Teleport in FIPS 140-2 mode by default) have `-fips` after the version,
-before the `<arch>` component, e.g. `teleport-ent-(=teleport.version=)-fips-x86_64`.
+FIPS 140-2 compatible AMIs (which deploy Teleport in FIPS 140-2 mode by default) have the `-fips` suffix after `<arch>`.
```

Comment:
I think it would make sense to include an example as the current docs do so it's explicit to a user what the AMI name is.

Reply:
Done in 23b9f3a46c